warehouse/templates/includes/manage/manage-project-menu.html

```diff
@@ -45,6 +45,12 @@
       </a>
     </li>
     {% endif %}
+    <li>
+      <a href="{{ request.route_path('manage.project.settings.publishing', project_name=project.normalized_name) }}" class="vertical-tabs__tab vertical-tabs__tab--with-icon {% if active_tab == 'publishing' %}vertical-tabs__tab--is-active{% endif %} {% if mode == 'mobile' %}vertical-tabs__tab--mobile{% endif %}" {% if active_tab == 'publishing' %}aria-selected="true"{% endif %}>
+        <i class="fa fa-upload" aria-hidden="true"></i>
+        {% trans %}Publishing{% endtrans %}
+      </a>
+    </li>
```

Comment:
This should be behind the feature flag.

Reply:
[c2ca980](https://github.com/pypa/warehouse/pull/10753/commits/c2ca980fc4ae3551631d5434212b4b7a0d2f1085)